src/pages/management/changelog/index.mdx

```diff
@@ -37,6 +37,10 @@ The developer preview version provides a preview of the upcoming changes to the
 - The [**List Agents**](/management/configuration-api/v3.5/#list-agents) and [**Get Agent**](/management/configuration-api/v3.5/#get-agent) methods now return agent's `account_id`.
 - Added new additional `last_logout` agent field for methods [**List Agents**](/management/configuration-api/v3.5/#list-agents) and [**Get Agent**](/management/configuration-api/v3.5/#get-agent).
 
+### Properties
+
+- There's a new request, [List Groups Properties](/management/configuration-api/v3.5/#list_group_properties).
```

Comment:
I believe the link should finish with `/v3.5#list-groups-properties' as this one does not work 🤔

Reply:
I changed it to `[List Groups Properties](/management/configuration-api/v3.5/#list-groups-properties).` Note the slash after v3.5, it's not in your suggestion, but it's in every other link, so I _think_ it's the correct way to do this. If not, I can change it.

Reply:
I have checked on the preview, and it is working :)